vue/components/mock/router-link/router-link.vue

```diff
@@ -1,6 +1,9 @@
 <template>
     <div>
-        <slot>
+        <slot
+            v-bind:navigate="() => {}"
+            v-bind:href="'https://platforme.com?mock_router_link=true'"

```

Comment:
why this?

Reply:
To avoid warnings because of this:

![image](https://user-images.githubusercontent.com/3048875/77060873-3070c680-69d1-11ea-882f-b6a132e58d81.png)

Which is the proper usage for a real `router-link` implementation.